### PR TITLE
Feat/supabase funcs guide

### PR DESCRIPTION
**Pull request by jyecusch**

### CODE REVIEW

File: docs/guides/nitric-and-supabase.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+## Test your Function Hook
+
+You can now test that your Function Hook is working by adding a new record to the `profiles` table. An easy way to do that is with the Supabase Table Editor. Navigate to the _Table Editor_, select the _profiles_ table and click _Insert row_:
+
+<img alt="Supabase editor screenshot"
+src="../../assets/img/guides/nitric-and-supabase/supabase_insertrow.png"
+height="400"
+/>
+
+That it!
```
Comment (tjholm) [inline docs/guides/nitric-and-supabase.md]: That's it!

**Review by tjholm — CHANGES_REQUESTED** [review]
Just one typo, other than that looks good

File: docs/guides/nitric-and-supabase.md (hunk 1 above)
Comment (jyecusch) [reply docs/guides/nitric-and-supabase.md]: Fixed

**Review by davemooreuws — APPROVED** [review]
Pushed a change to ensure relative urls for docs. Looks good.

**Comment by raksiv** [comment]
Ran through the demo - looked good.

**Review by raksiv — APPROVED** [review]

**Pull request merged**